CHANGELOG.md

```diff
@@ -1,6 +1,28 @@
 # Release notes
 This page documents production updates to Cloud Tools for IntelliJ. You can check this page for announcements about new or updated features, bug fixes, known issues, and deprecated functionality.
 
+## 17.9.1
+
+### Added
+  - Added the ability to change the name of the staged artifact for App Engine flexible deployments.
+    [1610](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1610)
+
+### Changed
+  - App Engine flexible deployment configurations now default to deploy the artifact as-is, without
+    renaming to `target.jar` or `target.war`. [1151](https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/1151)
```

Comment:
pull -> issue (I think github redirects it for you, but would be nice to fix)

Reply:
Whoops, done. Also added changes to `plugin.xml`.